orangecontrib/geo/widgets/owgeotransform.py

```diff
@@ -40,107 +56,106 @@ class Outputs:
     attr_lon = settings.ContextSetting(None)
     from_idx = settings.Setting("Slovenia 1996 / Slovene National Grid")
     to_idx = settings.Setting("WGS 84")
+    replace_original = settings.Setting(True)
 
     want_main_area = False
     resizing_enabled = False
 
     EPSG_CODES = get_projections()
 
     class Error(OWWidget.Error):
-        no_lat_lon_vars = Msg(
-                "Data has no latitude and longitude variables.")
+        no_lat_lon_vars = Msg("Data has no latitude and longitude variables.")
 
     def __init__(self):
         super().__init__()
         self.data = None
 
-        box = gui.vBox(self.controlArea, box="Coordinates:")
-
+        layout = QFormLayout()
+        gui.widgetBox(self.controlArea, box="Coordinates:", orientation=layout)
```

Comment:
Using `QFormLayout` changes the alignment according to native look and slightly decreases the spacing.

Before:
<img width="340" alt="Screenshot 2022-02-02 at 18 09 00" src="https://user-images.githubusercontent.com/2387315/152225524-2262a3c6-1957-46e4-9943-4870d23920b4.png">

After:
<img width="337" alt="Screenshot 2022-02-02 at 18 08 41" src="https://user-images.githubusercontent.com/2387315/152225535-27ab760d-f696-4e39-bfdf-c12000035359.png">